### DISCUSSION TITLE
Does Auto Complete support opening the options on click and showing all options with selected option highlighted

### DISCUSSION THREAD

**Discussion by pavankjadda** [primefaces/community#2945] [PrimeNG]
I started play with PrimeNG Auto Complete.  I can not find 2 options in docs or examples

1. Support opening the options panel on click
2. Show all options with selected option highlighted like https://mui.com/material-ui/react-autocomplete/

These seem like a basic features and supported by all auto completes in general

**Comment by qwadrox** [comment]
@pavankjadda 
I think you're looking forward to this option: https://primeng.org/autocomplete#dropdown

<img width="865" alt="image" src="https://github.com/user-attachments/assets/8c9da59a-0d67-407b-b780-c39bfc7762da">

**Reply by pavankjadda** [reply]
> @pavankjadda I think you're looking forward to this option: https://primeng.org/autocomplete#dropdown
> 
> <img alt="image" width="865" src="https://private-user-images.githubusercontent.com/33902750/384004697-8c9da59a-0d67-407b-b780-c39bfc7762da.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzA5OTUzNzIsIm5iZiI6MTczMDk5NTA3MiwicGF0aCI6Ii8zMzkwMjc1MC8zODQwMDQ2OTctOGM5ZGE1OWEtMGQ2Ny00MDdiLWI3ODAtYzM5YmZjNzc2MmRhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDExMDclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMTA3VDE1NTc1MlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWMzMmQ3ZWRhZjQ5N2NiNThkNmYzYTdlZDYzNDBmYTQwMjQxODM2MmQ1Mzc3MzU4Y2M5NGRhNTI0N2JmNmU2NTYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.CL10huZbzii01dGS65ZH2G9I-StPhViCn8LxG4kLE5E">

I tried. It doesn't work as expected.

**Reply by qwadrox** [reply]
Well without further information, screenshot and example app the community and the mainteners won't be able to help you.
